### DISCUSSION TITLE
Interface feature

### DISCUSSION THREAD

**Discussion by ghost** [codecadwallader/codemaid#824] [Ideas]
I keep running into instances where I would like to know if a Class that implements and Interface has additional methods.  There have been times (probably in a hurry) where the signature to a method should have been inserted into the Interface first then implemented in the concrete class but instead was just put into the concrete class.  

As part of cleanup/refactor it would sure be nice to see all of the method signatures in classes that implement interfaces where the method signature is NOT a signature in the Interface it implements.  

Sure there will normally always be signatures in concrete classes that are not in the interface but hunting and pecking to find them is very time consuming in large projects.  

Can't wait for the VS2022 support!  I'm missing CodeMaid!

**Comment by codecadwallader** [comment]
Thanks for the suggestion!  One peculiar idea that comes to mind is that if you utilized explicit interface implementations (e.g. `ISomeInterface.MethodName` vs. `MethodName`) then it might be more obvious which functions _aren't_ a part of an interface.  There's some other consequences though (e.g. you have to strip out access modifiers) so it may not be a very good option, but thought I'd share it since it came to mind.
